MapboxDirections/MBRouteOptions.swift

```diff
@@ -319,6 +334,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
             params.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
         }
         
+        if let locale = locale, var languageCode = (locale as NSLocale?)?.object(forKey: .languageCode) as? String, supportedLocales.contains(languageCode) {
+            if languageCode == "zh" {
+                languageCode = "zh-Hans"
+            }
+            params.append(URLQueryItem(name: "language", value: languageCode))
```

Comment:
Once we implement fallback logic on the server side, we’ll want to pass in `locale.localeIdentifier` here.

Reply:
Updated.